share/spice/statista/statista.js

```diff
@@ -0,0 +1,88 @@
+(function (env) {
+    "use strict";
+    
+    function getImage(item, size, blank) {  
+       if (item.teaserImageUrls[size]) {
+            var img = item.teaserImageUrls[size].src;
+            if (blank == 1) {
+                img = img + '?blank=blank';
+            }
+            return img;
+        } else if (size == 1) {
+            return 'http://statistacloudfront.s3.amazonaws.com/Statistic/table/table-355-1.png';
+        } else if (size == 2) {
+            return 'https://static1.statista.com/Statistic/table/table-100-1.png';
+        }
+    }
+    
+    function getTitle(title) {
+        return title.replace(/\ \|\ .+?$/, "");
+    }
+    
+    function formatDate(date) {
+        var matches = date.match(/(\d+)\.(\d+)\.(\d+)/);
+        var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+        return months[(parseInt(matches[2])-1)] + ' ' + matches[3];
+    }
+    
+    function getPremiumText(item) {
+        if (item.Premium == 1) {
+            return 'Premium';
+        } else {
+            return 'Free';
+        }
+    }
+    
+    function getPremiumCssClass(item) {
+        if (item.Premium == 1) {
+            return 'premium';
+        } else {
+            return 'free';
+        }
+    }
+    
+    env.ddg_spice_statista = function(api_result){
+
+        if (!api_result || api_result.error) {
+            return Spice.failed('statista');
+        }
+
+        Spice.add({
+            id: "statista",
+            name: "Statista",
```

Comment:
Let's change this to "Statistics" our policy is to not show brand names as tabs, instead we categories/topics from out set list.